src/java.base/share/classes/java/lang/invoke/VarHandles.java

```diff
@@ -629,6 +628,13 @@ private static void noCheckedExceptions(MethodHandle handle) {
                     noCheckedExceptions((MethodHandle) arg);
                 }
             }
+        } else {
+            final Class<?>[] exceptionTypes = MethodHandles.checkedExceptions(handle);
```

Comment:
Add a comment here to clarify that this is temporary and shouldn't be ported forward and I'm OK with it.

```suggestion
            /* Temporary code to handle OpenJ9's MethodHandle implementation.  This will be removed in a future release */
            final Class<?>[] exceptionTypes = MethodHandles.checkedExceptions(handle);
```

Reply:
Added the comment.